app/selectors/transaction.js

```diff
@@ -0,0 +1,8 @@
+import { reduce } from 'lodash';
+import { createSelector } from 'reselect';
+
+const getTransactionItem = (state) => (state.transaction_item);
+
+export const transactionAmountSelector = createSelector(getTransactionItem, (items) => (
+  reduce(items, (memo, item) =>(memo + item.value * item.quantity), 0)
+));
```

Comment:
@hotay Also there is not much benefit in using reselect as the state tree is not large. Please try to use the basic stuffs in react and redux, we want to ensure we don't have loads of dependencies and also make it easier for other developers to work on the project.